utils/concatAlbumDescription.ts

```diff
@@ -0,0 +1,82 @@
+import { Album } from "@/schemas/collections.schemas";
+
+export default function concatAlbumDescription(album: Album) {
```

Comment:
I would just accept the formats array, you don't need to send the full album here. Maybe rename the function `getFormatsDescription` or something